spot-ml/src/main/scala/org/apache/spot/dns/model/DNSSuspiciousConnectsModel.scala

```diff
@@ -205,15 +201,16 @@ object DNSSuspiciousConnectsModel {
                        userDomain: String,
                        url: String): TempFields = {
 
-    val DomainInfo(_, topDomainClass, subdomain, subdomainLength, subdomainEntropy, numPeriods) =
+    val DomainInfo(_, topDomainClass, _, subDomainLength, subDomainEntropy, numPeriods) =
```

Comment:
Seems like subdomain was not being used at all. At least that's what IntelliJ was telling me.